Easydict/App/Localizable.xcstrings

```diff
@@ -2335,6 +2383,22 @@
         }
       }
     },
+    "remove_code_comment_symbols" : {
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Remove code comment symbols「/*#」"
```

Comment:
Could we omit the `「/*#」` and `key_value —> key value`  part to make these two menu items look shorter?

Reply:
> Automatically remove code comment symbols「/*#」

I think it's ok, since we have already displayed it completely in the settings.

Reply:
Yes, another one is I am not sure if using `Remove comment symbols` is enough here？